docs/security/lcmaps-voms-authentication.md

```diff
@@ -165,6 +165,50 @@ The program edg-mkgridmap (found in the package `edg-mkgridmap`), used for authe
         In the output from this command, yum should **not** list other packages than the one.
         If it lists other packages, cancel the erase operation, make sure the other packages are updated to their latest OSG 3.4 versions (they should have ".osg34" in their versions), and try again.
 
+#### Migrating from GUMS
+
+GUMS is no longer available starting in OSG 3.4 and is being replaced by the LCMAPS VOMS plugin.
+Note that unlike GUMS, which runs on a central host, the LCMAPS VOMS plugin will run on your GUMS clients (e.g. HTCondor-CE, GridFTP, and XRootD).
+To migrate any custom configuration from GUMS to the LCMAPS VOMS plugin, perform the following procedure:
+
+1. On your GUMS host, retrieve the conversion helper script and run it:
+
+        :::console
+        root@gums-host # wget https://raw.githubusercontent.com/opensciencegrid/osg-vo-config/mapfile-generator-0.1/bin/manual-mapfile-from-gumsdb.py
+        root@gums-host # python manual-mapfile-from-gumsdb.py
```

Comment:
So are you supposed to redirect this output somewhere?

I'm not a big fan of having users copy output from the terminal. This should go into a file and users should review the file instead.

I also don't like including output in the docs since it gets outdated too quickly. You should have a few example lines instead.